.devcontainer/postCreate.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+python -m venv .venv
+source .venv/bin/activate
+python -m pip install -r requirements.txt
+python -m pip install -r requirements-dev.txt
+pre-commit install
+pre-commit run --all-files
+python -m pip install -e .
+
+# Installing during postCreateCommand instead of at docker creation
+# because foundryup installs the binaries in place not accessible by the
+# current (vscode) user.
+deactivate
+mkdir foundry
+cd foundry
```

Comment:
I don't think we need these, no files are created in this foundry dir right?

Reply:
yep, leftover from a different way of doing this, removed